.gitignore

```diff
@@ -33,5 +33,4 @@ yarn-error.log*
 # vercel
 .vercel
 
-public/wp-search.json
-public/feed.xml
\ No newline at end of file
+/public
```

Comment:
i would think we would still want to include other files inside public no? such as the favicon that's in there and any future files that gets added there. i think the goal with the search and feed were because they're autogenerated at build time

Reply:
side note: i'm also not opposed to putting these files in a completely separate directory instead of `public`. my original intent was to provide people the ability to make a clientside request to /wp-search.json if they wanted to

Reply:
actually - we need feed.xml and sitemap.xml to be in the /public directory no matter what so scratch that side note

Reply:
Sure! I'll change to ignore the files: feed.xml, sitemap.xml, wp-search.json and robots.txt